src/react-apps/applications/shared/src/navigation/drawer/TabletDrawerMenu.tsx

```diff
@@ -26,7 +26,8 @@ export interface ITabletDrawerMenuState {
   selectedMenuItem: string;
 }
 
-class TabletDrawerMenu extends React.Component<ITabletDrawerMenuProps, ITabletDrawerMenuState> {
+class TabletDrawerMenu extends React.Component<ITabletDrawerMenuProps & WithStyles<typeof Object>,
```

Comment:
`Object` is already a type, with a lower case `o` (or you're using JavaScripts primitive types), so no need for `typeof`.

Reply:
I changed it to typeof styles again as the materia-ui examples are using it. If i remove the typeof it's throwing error